docs/docsgen/source/api/checker.rst

```diff
@@ -5,6 +5,36 @@ onnx.checker
 .. contents::
     :local:
 
+check_value_info
++++++++++++
```

Comment:
The underline should have the same length.

Reply:
Updated. Thanks!